iwyu_include_picker.cc

```diff
@@ -1304,14 +1305,16 @@ vector<string> IncludePicker::GetCandidateHeadersForFilepathIncludedFrom(
     const string& included_filepath, const string& including_filepath) const {
   vector<string> retval;
   const string quoted_includer = ConvertToQuotedInclude(including_filepath);
```

Comment:
Because you said so? ;-) Don't know what to write there. I would have used its own path with the reasoning that this results in the filename only unless there is an `-I` that matches. W/O any path this will default to the absolute path.